.github/ISSUE_TEMPLATE/avm_module_issue.yml

```diff
@@ -60,9 +60,6 @@ body:
         - "avm/ptn/azd/container-app-upsert"
```

Comment:
@matebarabas I guess we'd also want to update the CHANGELOG.md? Or is that & the ownership kept as is?

Reply:
Updated the CHANGELOG.md files